course05/homework01/freekpaans/string.cc

```diff
@@ -0,0 +1,76 @@
+#include "string.h"
+
+#include <algorithm>
+#include <stdexcept>
+
+String::String()
+{
+}
+
+String::String(std::string s)
```

Comment:
s -> str? We dont like one char variable name much, except for the classic `for (int i`